meson.build

```diff
@@ -17,7 +17,9 @@ project(
 )
 
 cython = find_program('cython')
+pythran = find_program('pythran')
 generate_f2pymod = files('tools/generate_f2pymod.py')
+cythonize = files('tools/cythonize.py')
```

Comment:
Let's get rid of this line. In case the Cython support built into Meson is not sufficient yet, we can add a TODO - I really want to avoid using setuptools anywhere.